### DISCUSSION TITLE
Webviews ChomeOS 124?

### DISCUSSION THREAD

**Discussion by coolquiztestgenerator** [3kh0/ext-remover#1049] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Is there any webviews that work in ChromeOS 124? And if so, where can I find them?

**Comment by CJBrandi** [comment]
I don't really know. I would check other discussions related to web views though. One thing I do know is the one related to drone blocks is now patched.

**Reply by AshtonDavies** [reply]
> I don't really know. I would check other discussions related to web views though. One thing I do know is the one related to drone blocks is now patched. 

As the first to discover the DroneBlocks webview, I do know that the easy route was patched years ago before it was well known, so you're probably talking about the second one other people have found, but if that route is now patched, you may be able to find yet another if it's your only option, considering webview patches are usually pretty weak (for example, SpaceSaver2000 has worked around College Board patches multiple times I believe).

**Reply by CJBrandi** [reply]
> > I don't really know. I would check other discussions related to web views though. One thing I do know is the one related to drone blocks is now patched. 
> 
> As the first to discover the DroneBlocks webview, I do know that the easy route was patched years ago before it was well known, so you're probably talking about the second one other people have found, but if that route is now patched, you may be able to find yet another if it's your only option, considering webview patches are usually pretty weak (for example, SpaceSaver2000 has worked around College Board patches multiple times I believe).

Can you tell me instructions on how to do the second route because I believe I've tried every way

**Reply by AshtonDavies** [reply]
If the route you currently know was not incredibly fast, that is the second route.

I do not remember the steps, because that was years ago and I generally do not support webview bypasses anymore.

**Comment by Blobby-Boi** [comment, marked as answer]
List of webviews that I know of:

chrome://chrome-signin
Lego WeDo 2.0
SparkVUE
Geogebra Classic
S2Bot4Scratch
McMillan learning ebook
Droneblocks
myHomework student planner
3Dview
Coding with chrome
Mackinvia
Ipevo Whiteboard
TestNav

**Reply by LizzyEll** [reply]
Instructions for test nav?

**Reply by Blobby-Boi** [reply]
Do you have the kiosk app or the webstore app? Its different

**Reply by S-PScripts** [reply]
bruh i think the chromebooks at our school had 1 app that was able to be downloaded from the webstore... it was the lego one. but then they removed it and i never downloaded it so womp womp to me aaaaaaaaaa

**Reply by S-PScripts** [reply]
ACTUALLY I HAD TWO APPS. THIS SECOND ONE WAS A WHITEBOARD APP (NOT ONE IN LIST) BUT I NEVER FOUND ANYTHING IN IT. UNTIL NOW!!!!!!!!!!!!!!! I FOUND A NEW FUCKING WEBVIEW FOR THIS WHITEBOaRD!!!!!!!!!!!!!!!!

**Reply by SIMONHSCH** [reply]
How to do w/ Drone blocks ?

**Reply by LizzyEll** [reply]
either or @Blobby-Boi

**Reply by stuartcatt** [reply]
how do I do the chrome-signin

**Reply by S-PScripts** [reply]
Step 1. Go to chrome://chrome-signin
Step 2. Click ok on the bottom right corner
Step 3. In the Email text box put google@d11.org
Step 4. Click signin options
Step 5. Now click signin with github
Step 6. Click the github logo
Step 7. In the search box on the top Right type google and then click see more topics then you will see all the google links click www.google.com now boom unblocked Google.

**Reply by SIMONHSCH** [reply]
And with DroneBlocks ?

**Reply by Blobby-Boi** [reply]
do you have the webstore app or the play app

**Reply by Blobby-Boi** [reply]
If you are using the play app it wont work 
If you are using the webstore app then you can find some steps online

**Reply by SIMONHSCH** [reply]
I don't have the play app but I have the webstore app.

**Reply by Blobby-Boi** [reply]
ok good. Then you can probably search up some steps (I would just type it but too hard)

**Reply by jeffrey6576i** [reply]
@Blobby-Boi did they patch droneblocks or smth? i been trying to figure out how to like use it

**Reply by AshtonDavies** [reply]
> did they patch droneblocks or smth? i been trying to figure out how to like use it

They've made it harder and harder to do.

**Reply by jeffrey6576i** [reply]
@AshtonDavies I can try? how do i do it i need intructions

**Reply by AshtonDavies** [reply]
@jeffrey6576i I don't remember, but it's been discussed already somewhere pretty sure.

**Reply by jeffrey6576i** [reply]
> @jeffrey6576i I don't remember, but it's been discussed already somewhere pretty sure.

alr i will try to find it

**Reply by jeffrey6576i** [reply]
@AshtonDavies yeah I couldn't find it
